src/main/resources/CodyBundle.properties

```diff
@@ -183,5 +183,9 @@ gotit.autocomplete.message=This is how Cody displays autocomplete suggestions.<b
 
 # Other Actions
 action.cody.restartAgent.text=Restart Cody Agent
-ignore.action-in-ignored-file.detail=Cody has ignored this file because of your Sourcegraph admin policy
-ignore.action-in-ignored-file.learn-more-cta=Learn more
+ignore.action-in-ignored-file.detail=This file has been ignored by an admin. Autocomplete, commands, and other Cody features are disabled.
+ignore.action-in-ignored-file.learn-more-cta=Learn about Cody Ignore
+ignore.action-in-ignored-file.title=Cody is Limited
```

Comment:
"Cody assistance is disabled"?

Reply:
Same reason as above... UX and PM have 👍  these

Reply:
Sounds like the label may change. Still, would like to get this landed. It is easy enough to update the string later.